bfd/src/main/java/gov/cms/ab2d/bfd/client/BFDClientImpl.java

```diff
@@ -62,43 +80,67 @@ public BFDClientImpl(IGenericClient bfdFhirRestClient) {
             exclude = { ResourceNotFoundException.class }
     )
     public Bundle requestEOBFromServer(String patientID) {
-        return
-                fetchBundle(ExplanationOfBenefit.class,
-                        ExplanationOfBenefit.PATIENT.hasId(patientID));
+        return client.search()
+                .forResource(ExplanationOfBenefit.class)
+                .where(ExplanationOfBenefit.PATIENT.hasId(patientID))
+                .count(pageSize)
+                .returnBundle(Bundle.class)
+                .execute();
```

Comment:
Curious of the reason for this change. This method does not seem to be related to the story.